content/en/docs/concepts/configuration/secret.md

```diff
@@ -25,78 +24,92 @@ is safer and more flexible than putting it verbatim in a
 ## Overview of Secrets
 
 A Secret is an object that contains a small amount of sensitive data such as
-a password, a token, or a key.  Such information might otherwise be put in a
-Pod specification or in an image; putting it in a Secret object allows for
-more control over how it is used, and reduces the risk of accidental exposure.
+a password, a token, or a key. Such information might otherwise be put in a
+Pod specification or in an image. Users can create secrets and the system
+also creates some secrets.
 
-Users can create secrets, and the system also creates some secrets.
+To use a secret, a Pod needs to reference the secret.
+A secret can be used with a Pod in two ways:
 
-To use a secret, a pod needs to reference the secret.
-A secret can be used with a pod in two ways: as files in a
+- As files in a
 {{< glossary_tooltip text="volume" term_id="volume" >}} mounted on one or more of
-its containers, or used by kubelet when pulling images for the pod.
+its containers.
+- By the kubelet when pulling images for the Pod.
 
 ### Built-in Secrets
 
-#### Service Accounts Automatically Create and Attach Secrets with API Credentials
+#### Service accounts automatically create and attach Secrets with API credentials
 
 Kubernetes automatically creates secrets which contain credentials for
-accessing the API and it automatically modifies your pods to use this type of
+accessing the API and automatically modifies your Pods to use this type of
 secret.
 
 The automatic creation and use of API credentials can be disabled or overridden
-if desired.  However, if all you need to do is securely access the apiserver,
+if desired.  However, if all you need to do is securely access the API server,
 this is the recommended workflow.
 
-See the [Service Account](/docs/tasks/configure-pod-container/configure-service-account/) documentation for more
-information on how Service Accounts work.
+See the [Service Account](/docs/tasks/configure-pod-container/configure-service-account/)
```

Comment:
One optional tweak if you think it helps:
```suggestion
See the [ServiceAccount](/docs/tasks/configure-pod-container/configure-service-account/)
```
(and maybe write “service account” in all-lowercase on the next line?

Reply:
👍